.github/PULL_REQUEST_TEMPLATE/control_plane_template.md

```diff
@@ -1,5 +1,11 @@
 # ARM (Control Plane) API Specification Update Pull Request 
 
+## PR review workflow diagram
+
+Please understand this diagram before proceeding. It explains how to get your PR approved & merged.
+
+![diagram](spec_pr_review_workflow_diagram.png)
+
```

Comment:
pl change it to the one attached to the announcement in the teams channel. That has a minor change to address the feedback from Ray chen on the email thread